database/postgres/methods/update/reviewsummary.js

```diff
@@ -0,0 +1,13 @@
+const client = require('../../index.js');
+
+const updateRating = async (currentData, newRating) => await client.query(`UPDATE review_summaries SET rating_${newRating} = rating_${newRating} + 1, rating_${currentData.review_rating} = rating_${currentData.review_rating} - 1 WHERE product_id = ${currentData.product_id}`);
```

Comment:
I'm not understanding why you're decrementing the review rating here:
`rating_${currentData.review_rating} = rating_${currentData.review_rating} - 1`

Reply:
I'm decrementing that rating because it is the old review rating, to be replaced with newRating